src/app/action-playground/action-playground.component.html

```diff
@@ -97,3 +97,11 @@
 
 
 <span dspGnd [gnd]="'(DE-588)119166895'"></span>
+
+<hr>
+
+<p>External links Directive</p>
+
+<a href="https://dasch.swiss/" class="linkDasch">DaSCH website</a><br>
```

Comment:
`class="link"` is not needed for `a`-tags

and on line 107 it's defined twice

Reply:
I added this class for the unsuccessful e2e tests. I will remove it or rename it.